pages/stable-version.html

```diff
@@ -264,7 +264,6 @@ <h5>Quickstart Installation Instructions</h5>
 
                 <p>Installation of Uyuni Proxy is done via the Uyuni Server web interface.</p>
                 <p>See the <strong>Installation Guide</strong> for step-by-step instructions for installing and configuring Uyuni Proxy.</p>
-                <p>The repository for the Proxy is <a href="https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable/images/repo/Uyuni-Proxy-POOL-x86_64-Media1/">https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable/images/repo/Uyuni-Proxy-POOL-x86_64-Media1/</a>
```

Comment:
given that  you dropped the  repo,  shouldn't make  sense to  also rephrase the other  paragraph before?

Reply:
How so? The previous paragraph tells you to check the install guide?

Reply:
I was just wondering  if we want  to link to the install  guide. but actually,  we  could  also leave  it  as  it is.

Reply:
Let leave it this way for now. The guides are linked above, so it should be OK for the moment.